src/algebra/group_action_hom.lean

```diff
@@ -137,6 +150,24 @@ variables {M A B}
 theorem ext_iff {f g : A →+[M] B} : f = g ↔ ∀ x, f x = g x :=
 ⟨λ H x, by rw H, ext⟩
 
+instance : has_zero (A →+[M] B) :=
+⟨{ map_smul' := by simp,
+   .. (0 : A →+ B) }⟩
+
+instance : has_one (A →+[M] A) :=
+⟨{ map_smul' := by simp,
+   .. add_monoid_hom.id A }⟩
```

Comment:
If you defined this further down the file you'd have `id` available

Reply:
OK I've added https://github.com/leanprover-community/mathlib/pull/7847/commits/1322b50f77d47cc82aab052da43a68cea08cd0df to use `id`.